src/api/spec/models/project_find_package_spec.rb

```diff
@@ -23,15 +23,15 @@
       let(:package) { create(:package, project: project) }
 
       it 'returns the package from the link' do
-        expect(subject.project).equal?(link_target)
+        expect(subject.project).equal?(link_target) # rubocop:disable RSpec/MissingExpectationTargetMethod FIXME
```

Comment:
Why is this one not fixed?

Reply:
This is a false positive. `subject.project` and `link_target` are in fact different. However, the check `expect(subject.project).equal?(link_target)` doesn't fail.

Reply:
Alright